std/getopt.d

```diff
@@ -285,7 +285,7 @@ void main(string[] args)
         list.
 
 ---------
-void main(string[] args)
+int main(string[] args)
```

Comment:
In addition to what @JakobOvrum said, it's typical for `main` in D programs to just be `void` and use `return;`, so there's no need to change this line or line 300